packages/api/src/command/webhook/webhook.ts

```diff
@@ -217,3 +217,8 @@ export const sendTestPayload = async (url: string, key: string): Promise<boolean
   if (res.pong && res.pong === ping) return true;
   return false;
 };
+
+export const isDisableWH = (meta?: unknown): boolean => {
```

Comment:
nit: a better name could be `webhookIsDisabled`, will be easier to lookup in the future, and the caller doesn't need to know what `WH` means

Reply:
Agree that webhook is preferrable, but I suggest we keep `is` as prefix - has this on the previous team as a practice, boolean consts/variables start with `is` or `has`, it helps to quickly identify that's a boolean container.